screens/LoadingScreen.js

```diff
@@ -7,13 +7,24 @@ import { AppLoading } from "expo";
 import { Container, Content, View } from "../components/layout";
 import { Text } from "../components/typography";
 var jwtDecode = require("jwt-decode");
+import { Notifications } from "expo";
 import { tokenRedirect, setLocalLocation } from "../store/actions";
 import { connect } from "react-redux";
 import { getDeviceLocation } from "../utils/index";
 
 const LoadingScreen = props => {
   const fontsLoaded = false;
 
+  useEffect(() => {
+    this._notificationSubscription = Notifications.addListener(
```

Comment:
Now that I think about it there should probably be an unsubscriber on ```logout```

Reply:
I dont think this is necesssary. It seems like Expo is doing the event subscription thing for Notifications a little differently. Usually there would be a memory leak error if there are subscriptions on unmounted components.